build.xml

```diff
@@ -204,16 +264,30 @@
         </exec>
     </target>
 
-    <target name="apply-patches" if="has-patches">
-        <echo>${patches}</echo>
+
+    <target name = "git-patch" if="${has-git-patches}" >
+        <echo>${git-patches-list}</echo>
         <exec executable="git" failonerror="true">
             <arg value="apply"/>
-            <arg value="--directory=netbeans"/>
+            <arg value="--directory=${git-dir}"/>
             <arg value="--whitespace=nowarn"/>
-            <arg line="${patches}"/>
+            <arg line="${git-patches-list}"/>
         </exec>
     </target>
 
+    <target name="apply-patches">
+        <ant target="git-patch">
+            <property name="git-patches-list" value="${patches}"/>
+            <property name="has-git-patches" value="${has-patches}"/>
+            <property name="git-dir" value="netbeans"/>
+        </ant>
+        <ant target="git-patch">
+            <property name="git-patches-list" value="${patches-l10n}"/>
+            <property name="has-git-patches" value="${has-l10n-patches}"/>
+            <property name="git-dir" value="netbeans-l10n"/>
+        </ant>
+    </target>
+
     <target name="unapply-patches" if="has-patches">
```

Comment:
Please add similar changes for `unapply-patches` as done for `apply-patches` above.